packages/account-v2/package.json

```diff
@@ -5,6 +5,11 @@
     "engines": {
         "node": "18.x"
     },
+    "nx": {
+        "implicitDependencies": [
+            "@deriv/api"
```

Comment:
This makes sure that `account-v2` is only built when `@deriv/api` has already been run and build by nx

Reply:
I think due to the nature of `nx` parallel runs, the build orders are not consistent, thus `@deriv/api` may not be built yet before `account-v2` is run and symlinked to the dependent packages

Reply:
Why if we just put @deriv/api first in the build command before run-many 🙂 
![image](https://github.com/binary-com/deriv-app/assets/125247833/4b3dbf83-3986-489d-8d38-524287328ca5)

Reply:
hmm haven't tried that before 🤔